src/main/java/ByeCommand.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Class for the command to exit the bot.
+ * @author vanGoghhh
+ */
+
+public class ByeCommand extends Command {
+
+    /**
+     * Returns a string to indicate the exiting of the bot.
+     * @param tasks
```

Comment:
I think you can include more details on the params, such as the tasks are referring to the user inputted list?